React/tic-tac/src/redux/auth/actions.js

```diff
@@ -0,0 +1,14 @@
+function LogInTrue(payload) {
```

Comment:
Wrap both actions within the same object so you can access al "public" actions app-wide. 🙂